rcgen/src/crl.rs

```diff
@@ -196,39 +174,24 @@ pub struct CertificateRevocationListParams {
 }
 
 impl CertificateRevocationListParams {
-	fn serialize_der_with_signer(
-		&self,
-		issuer: &KeyPair,
-		issuer_name: &DistinguishedName,
-	) -> Result<Vec<u8>, Error> {
-		yasna::try_construct_der(|writer| {
-			// https://www.rfc-editor.org/rfc/rfc5280#section-5.1
-			writer.write_sequence(|writer| {
-				let tbs_cert_list_serialized = yasna::try_construct_der(|writer| {
-					self.write_crl(writer, issuer, issuer_name)?;
-					Ok::<(), Error>(())
-				})?;
-
-				// Write tbsCertList
-				writer.next().write_der(&tbs_cert_list_serialized);
-
-				// Write signatureAlgorithm
-				issuer.alg.write_alg_ident(writer.next());
-
-				// Write signature
-				issuer.sign(&tbs_cert_list_serialized, writer.next())?;
-
-				Ok(())
-			})
-		})
-	}
-	fn write_crl(
-		&self,
-		writer: DERWriter,
-		issuer: &KeyPair,
-		issuer_name: &DistinguishedName,
-	) -> Result<(), Error> {
-		writer.write_sequence(|writer| {
+	/// Serializes the certificate revocation list (CRL) in binary DER format, signed with
```

Comment:
```suggestion
	/// Serializes the certificate revocation list (CRL), signed with
```

Internally we use DER format but we return a wrapper around the DER format.